src/main/resources/META-INF/definitions.cnd

```diff
@@ -12,7 +12,7 @@
  - textColor (string, choicelist[resourceBundle]) = 'light' autocreated < 'light', 'dark'
  - subtitle (string) internationalized
  - body (string) internationalized
- - internalLink (weakreference, picker[type='editoriallink']) < 'jmix:droppableContent', 'jnt:page', 'jnt:contentFolder', 'jnt:folder'
+ - internalLink (weakreference, picker[type='pagesandfiles']) < 'jmix:droppableContent', 'jnt:page', 'jnt:contentFolder', 'jnt:folder'
```

Comment:
@wassek according to the [documentation](https://academy.jahia.com/documentation/knowledge-base/my-custom-picker-does-not-work-on-jahia-8), pagesandfiles is a custom picker that needs to be declared and added to the registry, I guess we're missing that part

Reply:
True.. It is defined in the bootstrap 4, but the dependency is to bootstrap 3. So I am not sure if the documentation is done well. Maybe we have to update the dependency?

Reply:
From what I understand, mention of bootstrap 4 is for the specific example in that case, but should also still apply to bootstrap 3 components

Reply:
I am not sure how we should proceed further? Should we add the picker? But then we would have it twice if you use bootstrap 4?